### PR TITLE
412 remove error symbol

### LINKED ISSUE #412: Bug of not showing the figures in README
Example:
https://github.com/Project-MONAI/model-zoo/tree/21d85dabfb38d013d15955ad9b1556f25bbdde3c/models/pancreas_ct_dints_segmentation

I did some experiments and it seems the bug comes from the `$` symbol.

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #412 .

### Description
This PR removes dollar symbols in readme in order to fix the figure unseen error..

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by Nic-Ma — APPROVED** [review]
@zephyrie Could you please help double confirm this PR will not affect your web page?

Thanks in advance.

**Comment by yiheng-wang-nv** [comment]
Since there are some other minor issues, I will push more commits within this PR to solve them.
The issues include:
1. typos in `pancreas_ct_dints_segmentation` readme
2. default value of pretrained flag in `pathology_tumor_detection` (This part needs @drbeh 's verify), also need to add print log restriction on multi gpu config. 
3. some bundles may need to describe `dataset_dir` in readme (needs discussion)

cc @Nic-Ma @KumoLiu

File: models/pancreas_ct_dints_segmentation/docs/README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ A neural architecture search algorithm for volumetric (3D) segmentation of the p
 ![image](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_net_arch_search_segmentation_workflow_4-1.png)
 
 ## Data
-The training dataset is the Panceas Task from the Medical Segmentation Decathalon. Users can find more details on the datasets at http://medicaldecathlon.com/.
+The training dataset is the Pancreas Task from the Medical Segmentation Decathalon. Users can find more details on the datasets at http://medicaldecathlon.com/.
 
-- Target: Liver and tumour
+- Target: Pancreas and pancreatic tumor
```
Comment (yiheng-wang-nv) [inline models/pancreas_ct_dints_segmentation/docs/README.md:9]: Hi @dongyang0122 , could you please help to double confirm the changes of this bundle? Thanks in advance!

File: models/pathology_tumor_detection/configs/train.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         "model_name": "resnet18",
         "num_classes": 1,
         "use_conv": true,
-        "pretrained": true
+        "pretrained": false
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/configs/train.json:35]: I did this change according to @KumoLiu 's comment on:
https://github.com/Project-MONAI/model-zoo/pull/357
Hi @drbeh , may need your double confirm here.

Comment (bhashemian) [reply models/pathology_tumor_detection/configs/train.json:35]: Thanks @yiheng-wang-nv, from development perspective, it doesn't matter and keeping it as `true` is fine (and more favorable for users). However, it is rather a product decision where we should see if we want to ship the pipelines to be able to be trained on ImageNet out-of-the-box or not, so I differ to @daguangxu and @pdogra89 for that.

PS: What is the situation with other pipeline? Are they using pretrained weights by default?

Comment (Nic-Ma) [reply models/pathology_tumor_detection/configs/train.json:35]: Hi @drbeh ,

For all the bundles which can use pretrained weights, I think we should set `pretrained=False`, because:
1. ImageNet is not for commercial use.
2. Our `model.pt` was trained by private dataset pretrained weights.

Thanks.

Comment (daguangxu) [reply models/pathology_tumor_detection/configs/train.json:35]: I also do not think it is critical. Either way is fine. Since all current bundle use False. Let's keep it simple here.

File: models/endoscopic_tool_segmentation/docs/README.md [hunk 3]
```diff
@@ -100,6 +100,12 @@ For more details usage instructions, visit the [MONAI Bundle Configuration Page]
 python -m monai.bundle run --config_file configs/train.json
 ```
 
+The default dataset path is defined as "/workspace/data/endoscopic_tool_dataset" in "configs/train.json", please modify it to the actual path via modifying `dataset_dir`:
+
+```
+python -m monai.bundle run --config_file configs/train.json --dataset_dir <actual dataset path>
+```
+
```
Comment (yiheng-wang-nv) [inline models/endoscopic_tool_segmentation/docs/README.md:108]: Hi @zephyrie , as for the `dataset_dir` notification, I added this part, could you please help to review it? If it's fine, I will adapt to all bundles.

Comment (yiheng-wang-nv) [reply models/endoscopic_tool_segmentation/docs/README.md:108]: cc @Nic-Ma

File: models/pancreas_ct_dints_segmentation/docs/README.md (hunk 1 above)
Comment (dongyang0122) [reply models/pancreas_ct_dints_segmentation/docs/README.md:9]: The adjustments made to the README are accurate. Thanks!

**Review by dongyang0122 — APPROVED** [review]

File: models/endoscopic_tool_segmentation/docs/README.md (hunk 3 above)
Comment (zephyrie) [reply models/endoscopic_tool_segmentation/docs/README.md:108]: @yiheng-wang-nv, that looks good!

File: models/pathology_tumor_detection/configs/multi_gpu_train.json [hunk 4]
```diff
@@ -28,7 +28,10 @@
         "$import torch.distributed as dist",
         "$dist.is_initialized() or dist.init_process_group(backend='nccl')",
         "$torch.cuda.set_device(@device)",
-        "$monai.utils.set_determinism(seed=123)"
+        "$monai.utils.set_determinism(seed=123)",
+        "$import logging",
+        "$@train#trainer.logger.setLevel(logging.WARNING if dist.get_rank() > 0 else logging.INFO)",
```
Comment (mingxin-zheng) [inline models/pathology_tumor_detection/configs/multi_gpu_train.json:33]: @yiheng-wang-nv I added a `RankFilter` for similar usage:

https://github.com/Project-MONAI/MONAI/blob/7f3d3fc79e0caa0953f801c903368e74a65a8773/monai/utils/dist.py#L182

Example to use
https://github.com/Project-MONAI/MONAI/blob/7f3d3fc79e0caa0953f801c903368e74a65a8773/monai/apps/auto3dseg/ensemble_builder.py#L560

Here it could be
```
"$@train#trainer.logger.addFilter(monai.utils.RankFilter())",
```

Please consider the suggestion as optional, and skip if needed.

Comment (yiheng-wang-nv) [reply models/pathology_tumor_detection/configs/multi_gpu_train.json:33]: Thanks Mingxin, in this version I suggest to not using this feature (require latest MONAI commit). Feel free to add a feature request and we can address it in next version.

**Review by Nic-Ma — APPROVED** [review]

File: models/endoscopic_tool_segmentation/docs/README.md (hunk 3 above)
Comment (yiheng-wang-nv) [reply models/endoscopic_tool_segmentation/docs/README.md:108]: Thanks @zephyrie , I did a bit change to the sentence, in order to match up with some bundles that are already mention the `dataset_dir` in readme.
